src/routes/setting/setting.js

```diff
@@ -28,6 +29,13 @@ const form = ({
     return rules
   }
   const genInputItem = (setting) => {
+    if (setting.id === 'volume-attachment-recovery-policy') {
```

Comment:
Instead of hardcoding this behavior for the volume-attachment-recovery-policy id, consider checking setting.definition.options. Than you can have a generic dropdown and the frontend is not dependent on the backend id or the option values.

Pseudo Code below
```
if (setting.definition.options) {
return 
(<Select>
        for each option : setting.definition.options {
                <Option value="$option">$option</Option>
        }
</Select>)
}
```

Reply:
Good catch @joshimoo 

@smallteeths can we do this instead?